lambdas/llama-2-13b-chat-llm/template.yml

```diff
@@ -0,0 +1,112 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: QnABot on AWS LLM Plugin for Llama-2-13b-chat
+
+Parameters:
+
+  LLMModel:
+    Type: String
+    Default: jumpstart-dft-meta-textgeneration-llama-2-13b-f
+    AllowedValues:
+      - jumpstart-dft-meta-textgeneration-llama-2-13b-f
+    Description: Llama 2 13b Chat LLM Model
+
+Resources:
+  LambdaFunctionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+      Policies:
+        - PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "sagemaker:InvokeEndpoint"
+                Resource:
+                  - "*"
+          PolicyName: SageMakerPolicy
+
+  LambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Handler: "llm.lambda_handler"
+      Role: !GetAtt 'LambdaFunctionRole.Arn'
+      MemorySize: 128
+      Timeout: 60
+      Runtime: python3.10       
+      Environment:
+        Variables:
+          ENDPOINT_NAME: !Ref LLMModel
```

Comment:
As noted above, param name is confusing, since you want the endpoint name, not the model. E.g. Call it `SageMakerEndpointName` as suggested earlier.

Reply:
Addressed in the 2nd commit.